inc/saiswitch.h

```diff
@@ -635,6 +635,86 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_EGRESS_BUFFER_POOL_NUM,
 
     /**
+     * @brief Available IPv4 routes
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV4_ROUTE_ENTRY,
+
+    /**
+     * @brief Available IPv6 routes
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV6_ROUTE_ENTRY,
+
+    /**
+     * @brief Available IPv4 Nexthop entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEXTHOP_ENTRY,
+
+    /**
+     * @brief Available IPv6 Nexthop entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV6_NEXTHOP_ENTRY,
+
+    /**
+     * @brief Available IPv4 Neighbor entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEIGHBOR_ENTRY,
+
+    /**
+     * @brief Available IPv6 Neighbor entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IPV6_NEIGHBOR_ENTRY,
+
+    /**
+     * @brief Available Next hop group member entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_NEXT_HOP_GROUP_MEMBER_ENTRY,
```

Comment:
Does the application want to know the total remaining members and not the members remaining per group ?